src/g3log/g3log.hpp

```diff
@@ -149,6 +154,14 @@ namespace g3 {
 #define CHECK(boolean_expression)        \
    if (false == (boolean_expression))  INTERNAL_CONTRACT_MESSAGE(#boolean_expression).stream()
 
+#ifdef ENABLE_WIN_WSTRING_SUPPPORT
+#	define LOGW(level) if(g3::logLevel(level)) INTERNAL_LOG_MESSAGE(level).wstream()
```

Comment:
1. Why would you define a different `LOGW` API if you anyways go through the wide char support in the LogMessage?

2. What does it give you to have access to both, when the `saveMessage` is changed?